acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -215,40 +215,6 @@ context('Acceptance tests', () => {
             cy.get('div[id=familienstand_zusammenveranlagung_field]').should('not.be.visible')
         })
 
-        it('Enter different tax number data ', () => {
```

Comment:
Why did you delete this test?

Reply:
As written in the description this test is now covered by the TaxNumberPage.tests. This was only the old way to test the tax number page